test_requirements.txt

```diff
@@ -6,8 +6,8 @@ ipdb
 nplusone>=0.8.1
 pdbpp
 pip-tools
-pylint==1.7.1
-pylint-django==0.7.2
+pylint==1.7.2
+ -e git+git://github.com/mitodl/pylint-django.git@2c7136f1cb3ff618aa08189c7fab4f8e08772eae#egg=pylint-django-0.7.2fix
```

Comment:
This is a forked version from 0.7.2 which cherry picks a bug fix for a rare recursive pylint error